packages/combobox/src/ComboboxContainer.tsx

```diff
@@ -43,6 +43,5 @@ ComboboxContainer.propTypes = {
 };
 
 ComboboxContainer.defaultProps = {
-  isAutocomplete: true,
```

Comment:
This wasn't making sense as the HOC element component is coming together. A consumer should knowingly opt into `aria-autocomplete` because they are expected to provide a corresponding option filtering mechanism.